src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -237,11 +238,17 @@ def nlcd_streams(sjs_result):
                          for count in (ag_count,
                                        low_urban_count,
                                        med_high_urban_count))
+    lu_streams = [0.0] * 16
+    for k, v in result.iteritems():
```

Comment:
Consider using `nlcd, streams` instead of `k, v` to clearly describe the contents of result.